sections/futures/Trade/MarketsDropdown.tsx

```diff
@@ -156,7 +144,7 @@ const MarketsDropdown: React.FC<MarketsDropdownProps> = ({ mobile }) => {
 					key: MarketKeyByAsset[marketAsset],
 					description: getSynthDescription(marketAsset, synthsMap, t),
 					price: mobile
-						? formatCurrency(selectedPriceCurrency.name, selectedBasePriceRate, {
+						? formatCurrency('sUSD', selectedBasePriceRate, {
```

Comment:
formatDollars?

Reply:
Ah. Always forgetting about that hook. Nice find.